web/Rakefile

```diff
@@ -79,7 +87,7 @@ all_to_copy.each do |dst, src|
   file dst => src do
     dstdir = File.dirname(dst)
     FileUtils.mkdir_p(dstdir) unless File.directory?(dstdir)
-    puts("cp #{src} #{dst}") if verbose
+    puts("cp #{src} #{dst}") if verbose == true
```

Comment:
Apparently Rake sets the value to the `:default` symbol so you need to explicitly check for `verbose == true`:
http://engineering.hoteltonight.com/checking-if-rake-was-called-with-verbose-or-t